src/Core/L10n.php

```diff
@@ -77,6 +77,10 @@ public static function setSessionVariable()
 				}
 			}
 		}
+		//if $_GET['lang'] is set force $_GET['lang']
```

Comment:
I'm guessing you meant `$_SESSION` for the second `$_GET`.

Reply:
Not 100% sure if you meant in the comment or in the code. But if you meant in the comment this is what I meant: if $_GET['lang'] is set force $_GET['lang'].

If you meant in the code. I'm setting the $_SESSION['language'] to $_GET['lang']. Because otherwise you're not forcing any language? Or did I drink too much again?